### PR TITLE
feat: Ory Summit

### PR DESCRIPTION
**Pull request by vinckr**
Adding 
annoucement banner
links in header/footer
event section in nav 
&
redirect. 

There might be some more redirects needed, but this should cover initial release.

### CODE REVIEW

File: src/page-content/navigation/navigation-header.tsx [hunk 1]
```diff
@@ -115,7 +115,7 @@ export const dropdownMenu: Pick<NavProps, 'dropdownMenu'> = {
     {
       title: (
         <Button to={'/jobs'} style={'link'}>
```
Comment (jfcurran) [inline src/page-content/navigation/navigation-header.tsx:117]: redirect to correct destination

File: src/page-content/navigation/navigation-announcement.tsx [hunk 2]
```diff
@@ -7,3 +7,10 @@ export const jobs: AnnouncementProps = {
   shortText: 'Ory is hiring!',
   link: '/jobs?mtm_campaign=job-banner'
 }
+
+export const summit: AnnouncementProps = {
+  longText:
+    'Sign up for Ory Summit 2021: free developer conference on security, IAM & more!',
+  shortText: 'Sign up for Ory Summit 2021!',
```
Comment (jfcurran) [inline src/page-content/navigation/navigation-announcement.tsx:14]: shortText to long

**Review by jfcurran — CHANGES_REQUESTED** [review]

**Review by jfcurran — APPROVED** [review]
okay nice, redirects are set correctly too

File: src/page-content/navigation/navigation-announcement.tsx [hunk 3]
```diff
@@ -7,3 +7,10 @@ export const jobs: AnnouncementProps = {
...
+  link: 'https://events.hubilo.com/ory-summit/register?mtm_campaign=ory-summit-2021&mtm_kwd=banner-landingpage'
```
Comment (zepatrik) [inline src/page-content/navigation/navigation-announcement.tsx:15]:
```suggestion
  link: '/summit21?mtm_campaign=ory-summit-2021&mtm_kwd=banner-landingpage'
```

File: src/page-content/navigation/navigation-header.tsx [hunk 4]
```diff
@@ -135,7 +135,7 @@ export const mobileMenu: Pick<NavProps, 'mobileMenu'> = {
           },
           {
             title: 'Pricing',
-            to: '/pricing'
+            to: '/pricing/'
```
Comment (zepatrik) [inline src/page-content/navigation/navigation-header.tsx:138]: why this change?

File: src/page-content/navigation/navigation-header.tsx [hunk 5]
```diff
@@ -114,8 +114,8 @@ export const dropdownMenu: Pick<NavProps, 'dropdownMenu'> = {
     },
     {
       title: (
-        <Button to={'/jobs'} style={'link'}>
-          Jobs
+        <Button to={'/summit21/'} style={'link'}>
```
Comment (zepatrik) [inline src/page-content/navigation/navigation-header.tsx:117]: Why the trailing slash
```suggestion
        <Button to={'/summit21'} style={'link'}>
```

File: src/page-content/navigation/navigation-header.tsx [hunk 6]
```diff
@@ -206,6 +206,15 @@ export const mobileMenu: Pick<NavProps, 'mobileMenu'> = {
             to: '/identity-in-modern-applications-oreilly/'
           }
         ]
+      },
+      {
+        category: 'Events',
+        mobileMenuItems: [
+          {
+            title: 'Ory Summit 2021',
+            to: '/summit21/'
```
Comment (zepatrik) [inline src/page-content/navigation/navigation-header.tsx:215]:
```suggestion
            to: '/summit21'
```

**Review by zepatrik — CHANGES_REQUESTED** [review]
Just some URLs. We should use `ory.sh/summit21` everywhere from now on, as that will allow us to drop in our own page at any point.

File: src/page-content/navigation/navigation-header.tsx (hunk 4 above)
Comment (vinckr) [reply src/page-content/navigation/navigation-header.tsx:138]: I was not sure if we should have trailing slashes or not, it seemed more had it than not, so I added it to be consistent. 
But they should be without right?
```suggestion
            to: '/pricing'
```

Comment (zepatrik) [reply src/page-content/navigation/navigation-header.tsx]: Don't see any with, probably both are handled but we should be consistent by not having them IMO.

File: src/page-content/navigation/navigation-header.tsx [hunk 7]
```diff
@@ -114,8 +114,8 @@ export const dropdownMenu: Pick<NavProps, 'dropdownMenu'> = {
     },
     {
       title: (
-        <Button to={'/jobs'} style={'link'}>
-          Jobs
+        <Button to={'/summit21'} style={'link'}>
+          Ory Summit
```
[the remark this answers is not in the document]
Comment (zepatrik) [reply src/page-content/navigation/navigation-header.tsx:118]: On mobile this is fine btw

**Review by zepatrik — APPROVED** [review]
Nice :+1:
